tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py3, docs, benchmarks, black, pylint
+envlist = py3, docs, benchmarks, black, isort, format, pylint
```

Comment:
Good idea for adding isort to the list of evs to run when calling `tox`.

However, I think we want to keep the default behaviour as just a check for correctness, which does not modify existing code, so I would prefer it if you could remove `format` from this list again.